world/client.cpp

```diff
@@ -816,27 +821,26 @@ bool Client::HandleEnterWorldPacket(const EQApplicationPacket *app) {
 			if (tutorial_enabled) {
 				zone_id = RuleI(World, TutorialZoneID);
 				database.MoveCharacterToZone(charid, zone_id);
-			}
-			else {
-				LogInfo("[{}] is trying to go to tutorial but are not allowed", char_name);
+			} else {
+				LogInfo("[{}] is trying to go to the Tutorial but they are not allowed.", char_name);
 				database.SetHackerFlag(GetAccountName(), char_name, "MQTutorial: player tried to enter the tutorial without having tutorial enabled for this character.");
 				eqs->Close();
 				return true;
 			}
 		}
 	}
 
-	if (zone_id == 0 || !ZoneName(zone_id)) {
+	if (!zone_id || !ZoneName(zone_id)) {
 		// This is to save people in an invalid zone, once it's removed from the DB
 		database.MoveCharacterToZone(charid, ZoneID("arena"));
-		LogInfo("Zone not found in database zone_id=[{}], moveing char to arena character:[{}]", zone_id, char_name);
+		LogInfo("Zone [{}] not found, moving [{}] to Arena.", zone_id, char_name);
 	}
 
-	if(instance_id > 0)
-	{
-		if (!database.VerifyInstanceAlive(instance_id, GetCharID()) ||
-		    !database.VerifyZoneInstance(zone_id, instance_id))
-		{
+	if (instance_id) {
+		if (
+			!database.VerifyInstanceAlive(instance_id, GetCharID()) ||
+		    !database.VerifyZoneInstance(zone_id, instance_id)
```

Comment:
This broke indent due to the space alignment before the second condition. (I also find this style a bit hard to read with the if scope parentheses newlines but that's just personal preference)